tests/python2/1_plus_1.exp

```diff
@@ -2,7 +2,7 @@
 
 spawn ./prybar-python2 -i
 match_max 100000
-expect "Python 2.*"
+#expect "Python 2.*"
```

Comment:
This seems to be the culprit for inconsistent tests. This regex seems to take a lot of time, at least in the ubuntu docker environment. So either we find a better way of expressing this expectation, leave it out, since it's not really testing any 1+1 specific details, or we check if there's some timeout we can set to > 30 seconds